src/root/providers/planning.providers.ts

```diff
@@ -1,5 +1,10 @@
 import { HttpClient } from '@angular/common/http';
 import { FactoryProvider } from '@angular/core';
 import { faresByDayRead$, faresByDayReadProvider } from '@features/planning';
+import { addFareToPlanningActionProvider } from '@features/planning/providers/actions/add-fare-to-planning.action.provider';
+import { addFareToPlanningAction$ } from '@features/planning/actions/create-fare/create-fare.action';
 
-export const PLANNING_PROVIDERS: FactoryProvider[] = [faresByDayReadProvider(faresByDayRead$, [HttpClient])];
+export const PLANNING_PROVIDERS: FactoryProvider[] = [
+  addFareToPlanningActionProvider(addFareToPlanningAction$, [HttpClient]),
+  faresByDayReadProvider(faresByDayRead$, [HttpClient])
```

Comment:
```suggestion
  addFareToPlanningProvider(addFareToPlanningAction$, [HttpClient]),
  faresByDayProvider(faresByDayRead$, [HttpClient])
```
Et si on partait du principe que si c'est un verbe ou ça commence par un verbe c'est une action, sinon c'est un read ?

Reply:
j'aime bien avoir action et read en explicite quand même

Reply:
OK, mais en me disant que je n'aimais pas voir `read`, je me suis dit que je n'aimais pas voir `action` non plus (même si c'est moins pire pour une raison que je ne peux pas expliquer)

Reply:
Est-ce qu'on peut laisser action et ne pas mettre `read` quand c'est un read, car on reçoit vraiment des `faresByDay` (sous forme d'observable) c'est un flux qui nous donne des fares by day => `faresByDay` + `$`

Quand il y a `action` ça met en emphase le fait qu'il y a un effet de bord, on ne reçoit pas de donnée, donc le nom ne peut pas ressembler à celui d'un modèle mais décrit plutôt l'action à produire

Reply:
👍 mais on le laisse quand même dans le nom du fichier pour la recherche